infra/cifuzz/fuzz_target.py

```diff
@@ -134,3 +186,67 @@ def get_test_case(self, error_string):
     if match:
       return os.path.join(self.out_dir, match.group(1))
     return None
+
+  def get_lastest_build_version(self):
+    """Gets the latest OSS-Fuzz build version for a projects' fuzzers.
+
+    Returns:
+      A string with the latest build version or None.
+    """
+    if not self.project_name:
+      return None
+    sanitizer = 'address'
+    version = '{project_name}-{sanitizer}-latest.version'.format(
```

Comment:
nit: put this string format into a constant.

Reply:
Done.